### PR TITLE
Create Ticket API structure

### PR DESCRIPTION
**Pull request by sk1384**

### CODE REVIEW

File: app/services/tickets/v1/create.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+		attr_accessor :title, :description, :ticket
+
+    def initialize(params)
+      @current_user = params[:current_user]
+      @title = params[:title]
+      @description = params[:description]
+    end
+
+		def call
+      save
```
Comment (varungadde99) [inline app/services/tickets/v1/create.rb:12]: change method name to - create_ticket

Comment (sk1384) [reply app/services/tickets/v1/create.rb:12]: @varungadde99 actually name of the service is create so function name is save

File: app/controllers/v1/tickets_controller.rb [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+module Api
+  module V1
+    class TicketsController < BaseController
```
Comment (varungadde99) [inline app/controllers/v1/tickets_controller.rb:5]: AppController

File: config/routes.rb [hunk 3]
```diff
@@ -2,5 +2,6 @@
   # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
   api_version(:module => "API::V1", :header => {
```
Comment (varungadde99) [inline config/routes.rb:3]: module => "Api::V1"

File: app/controllers/v1/tickets_controller.rb [hunk 4]
```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+module Api
```
Comment (varungadde99) [inline app/controllers/v1/tickets_controller.rb:3]: Folder structure - app/controller/api/v1/tickets_controller.rb

**Pull request closed without merging**